src/poke_env/player/player.py

```diff
@@ -262,6 +262,9 @@ async def _handle_battle_message(self, split_messages: List[List[str]]):
         :type split_message: str
         """
         # Battle messages can be multiline
+        will_move = False
+        from_teampreview_request = False
```

Comment:
i think these two could be named a bit more explicitly, for instance something like `should_process_request` and `should_process_teampreview_request`